plugins/ingest-useragent/build.gradle

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ 
+esplugin {
+  description 'Ingest processor that extracts information from a browser useragent'
+  classname 'org.elasticsearch.ingest.useragent.IngestUseragentPlugin'
+}
+
+dependencies {
+  compile 'commons-collections:commons-collections:3.2.2'
+}
+
+bundlePlugin {
+  from("${project.projectDir}/src/main/resources") {
+    into 'config/'
```

Comment:
Please don't do this. Plugins with static configuration files should be on their way out. This is a resource, and it should be loaded as a resource.